odoo/service/db.py

```diff
@@ -71,6 +72,9 @@ def _initialize_db(id, db_name, demo, lang, user_password, login='admin', countr
             if country_code:
                 country = env['res.country'].search([('code', 'ilike', country_code)])[0]
                 env['res.company'].browse(1).write({'country_id': country_code and country.id, 'currency_id': country_code and country.currency_id.id})
+                if len(country_timezones[country_code]) == 1:
```

Comment:
Are we sure we will find a match? Would probably be safer to use a `get` (if that works)

Reply:
Jep, `get` works, will just update.
```suggestion
                if len(country_timezones.get(country_code, [])) == 1:
```

Reply:
Couldn't we just do this?
`if  country_timezones.get(country_code):`